### DISCUSSION TITLE
Is there a way to change "language" url or options at runtime?

### DISCUSSION THREAD

**Discussion by habtista-peco** [l-lin/angular-datatables#1701] [Q&A]
Let say that I have a table with the following initialization options:

```
this.dtOptions = {
   ...,
   language: {
          lengthMenu: 'Show _MENU_ entries',
          infoEmpty: 'Showing 0 to 0 of 0 entries',
          paginate: {
             first: 'First',
             last: 'Last',
             next: 'Next',
             previous: 'Previous',
          },
      }
};

```
How can I change the dtOptions language.lengthMenu to 'Afficher _MENU_ entrées' and have the dataTable apply the changes?

**Comment by shanmukhateja** [comment]
Hi,

I think you will need to re-initialize the table  after changing `dtOptions.language.lengthMenu`. 

Please use the [Rerender](https://l-lin.github.io/angular-datatables/#/advanced/rerender) example to learn how to re-initialize the table.

**Reply by habtista-peco** [reply]
Hi,

Thanks for the quick reply.

I'm not sure if this applies to tables that are using "serverSide: true."

Can I use serverSide: true and a trigger on the same table?
